md/o61_may03_muscle.md

```diff
@@ -0,0 +1,78 @@
+กำหนดให้ $A_i$ แทนหมายเลขของนักกล้ามที่เดินเข้ายิมเป็นคนที่ $i$ และ $p_i$ คือจำนวนที่นักกล้ามตะโกนขณะที่นักกล้ามคนที่ $i$ กำลังเดินเข้ามาในยิม
+
+กำหนดให้ฟังก์ชั่น $L(S, x)$ เท่ากับจำนวนสมาชิกในเซต $S$ ที่มีค่าน้อยกว่าจำนวนเต็ม $x$
+
+ในข้อนี้ เราจะต้องหาลำดับของหมายเลขนักกล้ามที่เดินเข้ามาในยิม จากข้อมูลที่นักกล้ามแต่ละคนตะโกนเมื่อเดินเข้ายิม
+
+หากนักกล้ามคนที่ $1$ ถึง $n - 1$ ได้เดินเข้ามาในยิมเป็นที่เรียบร้อยแล้ว กำหนดให้ $S_i$ เป็นเซตของหมายเลขของนักกล้ามที่อยู่ในยิมแล้ว $i$ คนแรก จะได้ว่า $S_n$ เป็น $\{1, 2, ..., A_n - 1, A_n, A_n + 1, ..., n\}$ จะได้ว่า $p_n = |S_n| - L(S_n, A_n) - 1$ เนื่องจาก $|S_i| = i$ และในกรณีนี้ $L(S_n, A_n) = A_n - 1$ เสมอ ดังนั้นจะได้ว่า $p_n = n - A_n$ หรือ $A_n = n - p_n$ นั่นเอง
```

Comment:
- [ ] คำว่า "จะได้ว่า" ปรากฏสามครั้ง น่าจะแก้เป็นคำอื่นบ้าง (เช่น "ดังนั้น") หรือหาคำที่สื่อความหมายมากกว่านี้ (x implies y? x leads to thinking about y? etc.)
**Ctrl+F เช็คตรง paragraph อื่นด้วยนะครับ**

Reply:
paragraph อื่นไม่น่ามีคำซ้ำครับ แต่ลองเปลี่ยนบางคำให้สอดคล้องกับเนื้อหาดู